static/js/typeahead_helper.js

```diff
@@ -95,6 +95,29 @@ function prefix_sort(query, objs, get_item) {
 
 }
 
+function split_by_subscribers(people) {
+    var subscribers = [];
+    var non_subscribers = [];
+    var current_stream = compose.stream_name();
+
+    if (current_stream === "") {
+        // For the home page, everyone is a subscriber.
```

Comment:
(Ideally, for a group PM thread, we should just consider the subs to be the people who are in the PM recipients list).